cvat/apps/engine/models.py

```diff
@@ -305,6 +316,7 @@ class Shape(models.Model):
     occluded = models.BooleanField(default=False)
     z_order = models.IntegerField(default=0)
     points = FloatArrayField()
+    annotation_type = models.CharField(max_length=16, choices=AnnotationType.choices(), default="Manual", null=True)
```

Comment:
I suggest to define the `annotation_type` field in the annotation model, in this case it will be applicable for `tags` also.

Reply:
@azhavoro , 
Upon dumping and uploading annotations, the auto field changes to manual for all objects.

Reply:
I see, from my point of view adding of the `annotation type` field support for CVAT XML dumper/loader will be useful. What do you think about that?

Reply:
One quick question, adding the `annotation_type` field to cvat xml will include the field in the dumped .xml file or as in how will it be useful? If I were to add the field for CVAT XML dumper/loader I'm not sure as where to include it, should it be put here in `cvat/apps/annotation/models.py`

Reply:
> I suggest to define the `annotation_type` field in the annotation model, in this case it will be applicable for `tags` also.

Thanks for the suggestion, I tried to add the field under Annotation model, am I missing something here?
![Error](https://user-images.githubusercontent.com/59498234/76136974-a6823e80-605d-11ea-897f-9d3b412b5aa3.png)

Reply:
> I suggest to define the `annotation_type` field in the annotation model, in this case it will be applicable for `tags` also.

Thanks for the suggestions, added the field in Annotation model.

Reply:
> One quick question, adding the `annotation_type` field to cvat xml will include the field in the dumped .xml file or as in how is it useful? If I were to add the field for CVAT XML dumper/loader I'm not sure as where to include it, should it be put in here `cvat/apps/annotation/models.py`

One reason for including this info in the dumped .xml file is fully restorable task. For example: for some reason, I need to dump and upload annotation for the task, or I want to calculate some statistics based on annotations (percentage of  manually edited objects or smth else).

To add support of this field for `cvat xml` dumper/loader please take a look at https://github.com/opencv/cvat/blob/develop/cvat/apps/annotation/annotation.py and https://github.com/opencv/cvat/blob/develop/cvat/apps/annotation/cvat.py.